sdks/jreleaser-mastodon-java-sdk/src/main/java/org/jreleaser/sdk/mastodon/MastodonAnnouncer.java

```diff
@@ -52,9 +67,37 @@ public boolean isEnabled() {
 
     @Override
     public void announce() throws AnnounceException {
-        String status = mastodon.getResolvedStatus(context);
+        List<String> statuses = new ArrayList<>();
 
-        context.getLogger().debug("status: {}", status);
+        if (isNotBlank(mastodon.getStatusTemplate())) {
+            Map<String, Object> props = new LinkedHashMap<>();
+            props.put(Constants.KEY_CHANGELOG, MustacheUtils.passThrough(context.getChangelog()));
+            context.getModel().getRelease().getReleaser().fillProps(props, context.getModel());
+            Arrays.stream(mastodon.getResolvedStatusTemplate(context, props)
+                                 .split(System.lineSeparator()))
+                  .filter(StringUtils::isNotBlank)
+                  .map(String::trim)
+                  .forEach(statuses::add);
+        }
+        if (statuses.isEmpty() && !mastodon.getStatuses().isEmpty()) {
+            statuses.addAll(mastodon.getStatuses());
+            mastodon.getStatuses().stream()
+                   .filter(StringUtils::isNotBlank)
+                   .map(String::trim)
+                   .forEach(statuses::add);
+        }
+        if (statuses.isEmpty()) {
+            statuses.add(mastodon.getStatus());
+        }
+
+        for (int i = 0; i < statuses.size(); i++) {
+            String status = getResolvedMessage(context, statuses.get(i));
+            context.getLogger().info(RB.$("twitter.tweet"), status);
```

Comment:
This message resolves to "tweeting: {}" which is wrong for Mastodon. It should have its own resource message "mastodon.toot" resolving to "Tooting: {}"